### ISSUE TITLE
K factor should decrease over time

### ISSUE THREAD

**Issue by erikolsen** [erikolsen/elo_calculator#12]
Keeping the k factor at 50 make the variance too great to be representative of their actual skill after a certain number of games.  Maybe reduce k factor every 25 games with max reduction to 15?  Compare to US table tennis and US Chess Fed.

**Comment by ataz** [comment]
Agreed! Most implementations of ELO ranking systems that I've seen do this to some extent.
